Mage.Client/src/main/java/mage/client/draft/DraftPanel.java

```diff
@@ -62,12 +62,13 @@ public class DraftPanel extends javax.swing.JPanel {
      private List<String> setCodes;
 
      // Number of the current booster (for draft log writing).
-     // starts with 1
-     private int packNo;
+     private int packNo = 1;
```

Comment:
Can you elaborate on why you are now initializing these variables? Do they need to be referenced prior to when they were getting set before?

Reply:
```pickNo``` and ```timeoutPickNo``` are compared in ```loadBooster``` but they're set in ```updateDraft``` so this is to avoid an error in a scenario where someone rejoins mid-draft and gets ```loadBooster``` before ```updateDraft```, which I think is possible.